src/hooks/useHashLocation.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseLocationHook } from 'wouter'
+import { useCallback, useEffect, useState } from 'preact/hooks'
+
+// returns the current hash location in a normalized form
```

Comment:
Let's remove the comments, I think it's pretty self-explanatory what's going on here